taskt/Core/Automation/Commands/Data Commands/ParseJSONArrayCommand.cs

```diff
@@ -1,64 +1,64 @@
-﻿using System;
+﻿using Newtonsoft.Json.Linq;
+using System;
 using System.Collections.Generic;
-using System.Linq;
-using System.Xml.Serialization;
 using System.Data;
+using System.Linq;
 using System.Windows.Forms;
-using taskt.UI.Forms;
-using taskt.UI.CustomControls;
+using System.Xml.Serialization;
+using taskt.Core.Automation.Attributes.ClassAttributes;
+using taskt.Core.Automation.Attributes.PropertyAttributes;
+using taskt.Core.Automation.Engine;
+using taskt.Core.Script;
 using taskt.Core.Utilities.CommonUtilities;
+using taskt.UI.CustomControls;
+using taskt.UI.Forms;
 
 namespace taskt.Core.Automation.Commands
 {
     [Serializable]
-    [Attributes.ClassAttributes.Group("Data Commands")]
-    [Attributes.ClassAttributes.Description("This command allows you to parse a JSON Array into a list.")]
-    [Attributes.ClassAttributes.UsesDescription("Use this command when you want to extract data from a JSON object")]
-    [Attributes.ClassAttributes.ImplementationDescription("")]
+    [Group("Data Commands")]
+    [Description("This command parses a JSON array into a list.")]
     public class ParseJSONArrayCommand : ScriptCommand
     {
         [XmlAttribute]
-        [Attributes.PropertyAttributes.PropertyDescription("Supply the JSON Array or Variable (ex. {someVariable})")]
-        [Attributes.PropertyAttributes.PropertyUIHelper(Attributes.PropertyAttributes.PropertyUIHelper.UIAdditionalHelperType.ShowVariableHelper)]
-        [Attributes.PropertyAttributes.InputSpecification("Select or provide a variable or json array value")]
-        [Attributes.PropertyAttributes.SampleUsage("**[{obj1},{obj2}]** or **{vArrayVariable}**")]
-        [Attributes.PropertyAttributes.Remarks("")]
-        public string v_InputValue { get; set; }
+        [PropertyDescription("JSON Array")]
+        [InputSpecification("Provide a variable or JSON array value.")]
+        [SampleUsage("[{\"rect\":{\"length\":10, \"width\":5}}] || {vArrayVariable}")]
+        [Remarks("Provide only JSON Array data. Other data type will cause error.")]
```

Comment:
Change to "Providing data of a type other than a 'JSON Array' will result in an error."

Reply:
done